e2e/rook-test-framework/utils/k8s_helper.go

```diff
@@ -0,0 +1,245 @@
+package utils
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/jmoiron/jsonq"
+	"html/template"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+	"fmt"
+)
+
+type K8sHelper struct {
+}
+
+func CreatK8sHelper() *K8sHelper {
+	return &K8sHelper{}
+}
+
+func (k8sh *K8sHelper) getMonIp(mon string) (string, error) {
+	//kubectl -n rook get pod mon0 -o json|jq ".status.podIP"|
+	cmdArgs := []string{"-n", "rook", "get", "pod", mon, "-o", "json"}
+	out, err, status := ExecuteCmd("kubectl", cmdArgs)
+	if status == 0 {
+		data := map[string]interface{}{}
+		dec := json.NewDecoder(strings.NewReader(out))
+		dec.Decode(&data)
+		jq := jsonq.NewQuery(data)
+		ip, _ := jq.String("status", "podIP")
+		return ip + ":6790", nil
+	} else {
+		return err, errors.New("Error Getting Monitor IP")
+	}
+}
+
+func (k8sh *K8sHelper) CreatePodFromTemplate(poddefPath string) (string, error) {
+
+	t, _ := template.ParseFiles(poddefPath)
+	mons := k8sh.getMonitorPods()
+	ip1, _ := k8sh.getMonIp(mons[0])
```

Comment:
this retrieval of the mon IPs and inserting them into a pod spec template is no longer needed with @kokhang's new StorageClass and operator's volume provisioner, right?